common.blocks/radio-group/radio-group.bemhtml

```diff
@@ -27,6 +27,7 @@ block('radio-group')(
                     val : option.val,
                     text : option.text,
                     title : option.title,
+                    tabIndex : option.tabIndex || ctx.tabIndex,
```

Comment:
there is should not be separate tabIndex on options — just common tabIndex from parent